pages/linux/update-alternatives.md

```diff
@@ -0,0 +1,19 @@
+# update-alternatives
+
+> A convenient tool for maintaining symbolic links that determining default commands.
+
+- Add a symbolic link:
```

Comment:
Since all your examples are java based, we should mention this at the start. Then it will be implied. So I suggest - `Add a symbolic link for "java"`

Reply:
I didnt want you to remove all concrete examples with their generic counterparts :smile:. Those are fine. Just change the description to `Add a symbolic link for "java" with a priority of "300"`

Reply:
Yes, but I think a more generic expression maybe better. Or can we add a generic expression and a concrete examples

Reply:
We try to put concrete examples instead of generic ones so that beginners can have a better idea as to how the command is used. If you want, we can start with a generic example and then have concrete ones after that.